internal/utils/commit_stats.go

```diff
@@ -0,0 +1,366 @@
+package utils
```

Comment:
instead of a package `utils` which does not say much, what do you think about moving this to a `commitstats` package?

Reply:
inside of utils package?

Reply:
No, directly in internals

Reply:
More packages are about to come, will be a mess having many packages that the relation is being utils to the others.

Reply:
We're not gonna have that many packages. That's the very definition of internal: a collection of packages that are only meant for internal use by the package. Having an `utils` package does not say much other than the fact that the package is useful for some other package, which is why the package exists.